tools/README.md

```diff
@@ -0,0 +1,15 @@
+# Helpful Tools
+This directory contains a list of tools/scripts that is potentially helpful. 
```

Comment:
Nit: i think it should 'that are potentially helpful' since it's the tools/scripts not the list that you're talking about.

Reply:
good catch will fix.